.gitignore

```diff
@@ -1,3 +1,44 @@
+# Created by https://www.gitignore.io/api/windows,laravel
+
+### Windows ###
+# Windows image file caches
+Thumbs.db
+ehthumbs.db
+
+# Folder config file
+Desktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+# Windows Installer files
+*.cab
+*.msi
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
```

Comment:
Sorry for the changes in the ignore file. I didn't intend to include the file in the pull request at all. I thought it would be a good idea to have it in my base folder here on windows, so the github desktop application would ignore some typical windows files. You can just ignore this complete file if you like!